inlineplz/interfaces/github.py

```diff
@@ -92,3 +93,11 @@ def position(self, message):
                         if hunk_line.target_line_no == message.line_number:
                             return position + offset
                     offset += len(hunk) + 1
+
+    def clear_outdated_messages(self):
+        for comments in self.pull_request.review_comments():
+            if not comment.position and self.prefix in comment.body:
```

Comment:
maybe only run this if `self.prefix` is truthy? (in case someone wants to disable prefixes entirely)